commodore/component/template.py

```diff
@@ -88,8 +92,25 @@ def cookiecutter_args(self) -> dict[str, str]:
         args["add_lib"] = "y" if self.library else "n"
         args["add_pp"] = "y" if self.post_process else "n"
         args["add_matrix"] = "y" if self.matrix_tests else "n"
+        args["automerge_patch"] = "y" if self.automerge_patch else "n"
+        args["automerge_patch_v0"] = "y" if self.automerge_patch_v0 else "n"
         return args
 
+    @property
+    def automerge_patch(self) -> bool:
+        # TODO(sg): do we want this behavior?
```

Comment:
additional note: The template implementation requires that `automerge_patch` is set for `automerge_patch_v0` to have any effect.

Reply:
Feels like the expected, non-surprising behaviour to me.